drop-transfer/src/protocol/v5.rs

```diff
@@ -196,6 +198,7 @@ impl From<&OutgoingTransfer> for TransferRequest {
                 })
                 .collect(),
             id: value.id(),
+            canceling: false,
```

Comment:
This doesn't give me confidence... Should this field even be on this struct? It should probably not even reach this state if it's already "in cancel"

Reply:
Um, why? It was reached by the Android team and reported as a problem

Reply:
So the thing is that you want to know as soon as possible that the canceling is happening

Reply:
Yeah, but I am wondering if we should even send this xfer request if we know it's gonna be canceled. Elsewise we need to properly manage this `canceled` field which feels redundant if we already know it

Reply:
This is actually an interesting point - could we get away with just not communicating with the other side at all in this case? Since the whole reason for why Android wants this change is to not show anything on the UI, doubt they would need this info if they just want to instantly hide it

Reply:
@Lipt0nas do you suggest just skipping the transfer on the sender side altogether?
It's a valid option. However, in that case, you would have a mismatched history on both sides.
IDK, @LukasPukenis what do you think

Reply:
@matszczygiel Yeah, that's the idea

I see only benefits in this approach, considering with this we would allow hiding transfers from the UI, as a receiver, I wouldn't want these "ghost" transfers to appear in my history, on a very big stretch we could even call it abuse, if i'm able to fill up the transfer history with meaningless entries :D

Reply:
That was the original solution proposed by the Android team. I opted against it, because of history.
But I guess you managed to change my mind.
Let me implement this the way you described, moving the PR to draft mode